api/v1alpha1/nodepool_types.go

```diff
@@ -437,7 +476,7 @@ type PowerVSNodePoolPlatform struct {
 	// +kubebuilder:default=delete
 	// +kubebuilder:validation:Enum=delete;retain
 	// +optional
-	ImageDeletePolicy string `json:"imageDeletePolicy,omitempty"`
+	ImageDeletePolicy PowerVSNodePoolImageDeletePolicy `json:"imageDeletePolicy,omitempty"`
```

Comment:
why we haven't defined variables like `PowerVSNodePoolImageDeletePolicyDelete` or `PowerVSNodePoolImageDeletePolicyRetain`?

Reply:
Same reason as above.